docs/api.rst

```diff
@@ -820,11 +820,11 @@ Query the current price of assets
       {
           "assets": ["BTC", "ETH", "LINK", "USD", "EUR"],
           "target_asset": "USD",
-	  "ignore_cache": true,
+	      "ignore_cache": true
```

Comment:
here you need to change tabs to spaces. As you can see it does not render correctly